docs/send-data/hosted-collectors/amazon-aws/aws-sources.md

```diff
@@ -78,7 +78,7 @@ These configuration instructions apply to log collection from all AWS Source typ
 1. **Log File Discovery.** You have the option to set up Amazon Simple Notification Service (SNS) to notify Sumo Logic of new items in your S3 bucket. A scan interval is required and automatically applied to detect log files.
 
     :::important
-    Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-to-date Alerts.
+    Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-   to-date Alerts.
```

Comment:
Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-to-date Alerts.

Reply:
That space doesn't look correct.

Reply:
Ahh - it's definitely not. Typo on my end.